_includes/cards/audio.html

```diff
@@ -1,9 +1,30 @@
 <figure class="mt-4">
-  <header class="h4">Audio Testimonial</header>
+  {%- if page.audio-quote -%}
+  
+  <blockquote class="quote row">
+    <div class="col-md-2 quote-text quote-bg d-flex align-items-center justify-content-center">
+      <audio controls src="{{ page.audio }}" preload="none">
+        <a href="{{ page.audio }}"> Download audio </a>
+      </audio>
```

Comment:
@rpeterman-gp This is where I brought in the quote styling to the audio include...